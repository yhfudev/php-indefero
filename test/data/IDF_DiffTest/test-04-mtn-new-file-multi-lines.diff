#
# old_revision []
#
# add_dir ""
# 
# add_file "a"
#  content [9decd90bc2eedb78c74d5059a6bb6244dd2d0562]
#
============================================================
--- /dev/null	
+++ a	9decd90bc2eedb78c74d5059a6bb6244dd2d0562
@@ -0,0 +1,2 @@
+abc
+abc
