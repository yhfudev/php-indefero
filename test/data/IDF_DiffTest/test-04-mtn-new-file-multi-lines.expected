<?php return array (
  'a' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '',
          1 => '1',
          2 => "abc\r\n",
        ),
        1 => 
        array (
          0 => '',
          1 => 2,
          2 => "abc\r\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '0',
          1 => '0',
        ),
        1 => 
        array (
          0 => '1',
          1 => '2',
        ),
      ),
    ),
  ),
);